### DISCUSSION TITLE
能否手动更改内存限制？想增加内存的使用率

### DISCUSSION THREAD

**Discussion by mmm1h** [kkkgo/PaoPaoDNS#129] [问答和帮助] [answered]
如题，给paopao分配了6g的内存，实际的占用在1.5g左右，想了解paopao的自动根据内存调整机制是否可以更改为手动

**Comment by kkkgo** [comment, marked as answer]
你可以通过手动挂载`/proc/meminfo`来欺骗docker容器的可用内存。
比如编辑`/home/meminfo`:
```rust
MemAvailable 4500000
```
单位为KB。
`docker run -v/home/meminfo:/proc/meminfo sliamb/paopaodns`
在技术上，试图欺骗Docker如内存的信息可能会引起一些不可预见的稳定性和性能问题。
